examples/direct_fidelity_estimation.py

```diff
@@ -63,10 +63,11 @@ def compute_characteristic_function(circuit: cirq.Circuit,
     return trace, prob
 
 
-async def estimate_characteristic_function(
-        circuit: cirq.Circuit, pauli_string: cirq.PauliString,
-        qubits: List[cirq.Qid], simulator: cirq.DensityMatrixSimulator,
-        samples_per_term: int):
+async def estimate_characteristic_function(circuit: cirq.Circuit,
+                                           pauli_string: cirq.PauliString,
+                                           qubits: List[cirq.Qid],
+                                           sampler: cirq.Sampler,
```

Comment:
Args section of the docstring below needs to be updated.

Reply:
Done